nav2_amcl/include/nav2_amcl/amcl_node.hpp

```diff
@@ -354,6 +354,11 @@ class AmclNode : public nav2_util::LifecycleNode
    * @brief Publish particle cloud
    */
   void publishParticleCloud(const pf_sample_set_t * set);
+  /*
+  * @brief Get the current state estimate hypothesis from the particle cloud
+  * by calculating mean weighted centroid among cluster centroids
+  */
+  bool getWeightedMeanClustersCenroid(amcl_hyp_t & mean_centroid_hyp);
```

Comment:
"Cenroid", should be "Centroid"
Also the description says "mean weighted", but function name is "weighted mean"

Reply:
fixed